content/SCALE/SCALEUIReference/ReportingScreensSCALE.md

```diff
@@ -19,7 +20,7 @@ The **CPU** report displays by default.
 To configure a third-party reporting integration, such as Graphite, click **Exporters** to open the [**Reporting Exporters**](#reporting-exporters-screen) screen.
 
 ## Report Graphs
-The following sections provide examples of each report graph.
+The following sections provide examples of each graph produced through data reporting.
```

Comment:
The graph is a form of data reporting but it is produced from massaging data collected from the system so this is not really correct.
```suggestion
The following sections provide examples of each report graph.
```